src/cpp/session/modules/SessionRSConnect.R

```diff
@@ -13,6 +13,11 @@
 #
 #
 
+.rs.addJsonRpcHandler("forget_rsconnect_deployments", function(sourcePath, outputPath)
+{
+  rsconnect::forgetDeployment(sourcePath, force = TRUE)
```

Comment:
Nit: we almost always provide named arguments to rsconnect functions so that we can rearrange them in the package without worrying about breaking the IDE.